resources/views/components/general/entity-header-item.blade.php

```diff
@@ -14,7 +15,7 @@
 <div class="entity-header-item {{ $wrapperClass }}{{ $truncate ? ' overflow-x-auto truncate' : '' }}">
     @unless($withoutIcon)
         <div class="hidden items-center md:flex">
-            <div class="circled-icon text-theme-secondary-900 border-theme-secondary-900 dark:text-theme-secondary-600 dark:border-theme-secondary-600">
+            <div class="{{ $iconClass ?? 'circled-icon text-theme-secondary-900 border-theme-secondary-900 dark:text-theme-secondary-600 dark:border-theme-secondary-600' }}">
```

Comment:
doesn't seem to handle dark mode for migrations
<img width="289" alt="image (9)" src="https://user-images.githubusercontent.com/35610748/208149970-82005ee9-44cc-4576-b4ba-0679adf4fc67.png">